.circleci/config.yml

```
@@ -132,6 +133,20 @@ commands:
                   key: << parameters.apim-ui-project >>-cache-v1-{{ checksum "<< parameters.apim-ui-project >>/package-lock.json" }}
                   paths:
                       - ./<< parameters.apim-ui-project >>/node_modules
+    docker-scan:
+        steps:
+            - run:
+                name: Update image names
+                command: |
+                  export IMAGE="$IMAGE1:$IMAGE_TAG"
+                  echo "export IMAGE=$IMAGE" >> $BASH_ENV        
+            - snyk/scan:
+                organization: gravitee.io
+                monitor-on-build: true
+                severity-threshold: critical
+                docker-image-name: ${IMAGE}
+                fail-on-issues: false
+                project: ${IMAGE_TAG}
```

I think it's easier to understand and maintain if we are using parameters instead of environment variables, it simplify the calls also (see comments below):

```suggestion
    docker-scan:
        parameters:
            docker-image-name:
                type: string
                default: ""
                description: The name of the docker image to scan (without the version)
            version:
                  type: string
                  default: ""
                  description: Version of the docker image to scan
        steps:
            - snyk/scan:
                  organization: gravitee.io
                  monitor-on-build: true
                  severity-threshold: critical
                  docker-image-name: << parameters.docker-image-name >>:<< parameters.version >>
                  fail-on-issues: false
                  project: << parameters.version >>

```